Cargo.toml

```diff
@@ -15,7 +15,7 @@ chacha20poly1305 = "0.10.1"
 ed25519-dalek = { version = "2.1.0", features = ["rand_core"] }
 rand_chacha = { version = "0.3.1", optional = true }
 rand_core = "0.6.4"
-reddsa = { git = "https://github.com/ZcashFoundation/reddsa.git", rev = "311baf8865f6e21527d1f20750d8f2cf5c9e531a", features = ["frost", "frost-rerandomized"] }
+reddsa = { git = "https://github.com/ZcashFoundation/reddsa.git", rev = "b9c3107e6ec5333a89a7fa064f2d10f749a90cce", features = ["frost", "frost-rerandomized"] }
```

Comment:
Not sure if it's worth updating this to point to the proper commit here or in the follow up PRs - but does need to be done at some point

Reply:
I think we can defer until we merge in the next PR for the `no-std` branch since that also updates the commit for reddsa